docker/cron/Dockerfile

```diff
@@ -1,10 +1,8 @@
 FROM debian:latest
 
-RUN apt-get update && apt-get install -y \
-    cron \
-    rsyslog
+RUN apt-get update && apt-get install -y cron
```

Comment:
rsyslog writes log messages, these help us to check health of cron's jobs.
e.g. ( I did not mount /opt/backup/* files, so an  occurred error, that was logged)  
```bash
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 liblogging-stdlog:  [origin software="rsyslogd" swVersion="8.24.0" x-pid="13" x-info="http://www.rsyslog.com"] start
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 cron[23]: (CRON) INFO (pidfile fd = 3)
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 cron[24]: (CRON) STARTUP (fork ok)
se-backup-data | Jul  8 23:03:35 3acb1ea39ba1 cron[24]: (CRON) INFO (Skipping @reboot jobs -- not system startup)
se-backup-data | Jul  8 23:03:36 3acb1ea39ba1 liblogging-stdlog: imklog: cannot open kernel log (/proc/kmsg): Operation not permitted.
se-backup-data | Jul  8 23:03:36 3acb1ea39ba1 liblogging-stdlog: activation of module imklog failed [v8.24.0 try http://www.rsyslog.com/e/2145 ]
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 CRON[27]: (root) CMD (       sh /usr/bin/entrypoint.sh 2>&1 | logger)
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: database
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: /usr/app/src/database: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: media
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): /opt/backup/database-2018-07-08.tar.gz: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: /usr/app/src/media: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: static
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): /opt/backup/media-2018-07-08.tar.gz: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: /usr/app/src/static: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar: Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): /opt/backup/static-2018-07-08.tar.gz: Cannot open: No such file or directory
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: tar (child): Error is not recoverable: exiting now
se-backup-data | Jul  8 23:04:02 3acb1ea39ba1 root: find: '/opt/backup/*': No such file or directory
```

Why did you remove a log processing from this Dockerfile? How does it help us to resolve a permissions problem?